src/main/java/org/hydev/mcpm/client/Downloader.java

```diff
@@ -45,19 +45,22 @@
      */
     public void downloadFile(String url, File to)
     {
+        // TODO: Implement this
```

Comment:
You should delete these comments

Reply:
Is this your commit even 🤔

Reply:
I'm not too sure why these files were even added, I was playing around with download like over a week ago just to fix the bugs and I'm guessing I accidentally committed them.